apps/passport-client/src/broadcastChannel.ts

```diff
@@ -0,0 +1,46 @@
+import { Action } from "./dispatch";
```

Comment:
this file feels like the sort of thing that could live in `passport-interface` at some point in the future. just pointing it out, no suggestion here.

Reply:
Added a comment